module-2.md

```diff
@@ -10,3 +10,5 @@ ___
 
 
 [HYF-javascript1](https://github.com/abdulsalamalmahdi/HYF-javascript1), 
+
+https://github.com/yousefnk/HYF-javascript1
```

Comment:
Hi @yousefnk, could I ask you to update this link and the one above so we set a standard like :
```
[<first name>/HYF-javascript1](<link to repo>)
```
? If you commit to your `patch-1` branch it will update this PR. You could actually also edit directly from this view on GitHub as well.